<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginScreen</class>
 <widget class="QMainWindow" name="LoginScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginScreen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{border-image: url(:/images/kfc_2.png);}



QLineEdit{
		border-radius:20px;
		border:2px solid rgb(66, 66, 162);
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="SoftKeyBoard" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>188</y>
      <width>781</width>
      <height>259</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>300</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>160</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{
		border-radius:40px;
		background-color: rgba(100, 41, 41,100);
}
</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="accoEdit">
         <property name="minimumSize">
          <size>
           <width>145</width>
           <height>40</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>账号...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passEdit">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>密码...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loginBtn">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{border-radius:40px;}</string>
         </property>
         <property name="text">
          <string>登录</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SoftKeyBoard</class>
   <extends>QWidget</extends>
   <header location="global">softkeyboard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
